proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -332,18 +332,41 @@ private void loadPlugins() {
 
     try {
       Path pluginPath = Path.of("plugins");
+      Path updatePath = pluginPath.resolve("to-update");
```

Comment:
Personally, I am not convinced by the name `to-update` for the plugins folder to be updated. Also, if it were up to me, I would integrate the folder inside the plugins folder, since having 3 folders in the main Velocity directory dedicated to contain plugins would be very strange

Reply:
Im confused
Its already inside the plugin folder, I am doing `pluginPath.resolve()`